doc/user/content/integrations/cli/reference/sql.md

```diff
@@ -0,0 +1,41 @@
+---
+title: mz sql
+description: The `mz sql` command executes SQL statements in a region.
+menu:
+  main:
+    parent: cli-reference
+    weight: 1
+---
+
+The `mz sql` command executes SQL statements in a region.
```

Comment:
Are we wanting to swap from `mz shell` to `mz sql`? I think `mz shell` makes a bit more sense because it feels like a more common  terminology for this type of action 

(worth noting: I've spent a good bit of time using the [pscale cli](https://planetscale.com/docs/reference/shell), so it might be making me a little biased.)

Reply:
Ah, so the reason I switched it is because the command supports running one-off SQL commands (e.g. `mz sql -- -c "select 1"`), and that reads funny IMO with `mz shell`. Maybe `sql` is just exactly as bad in the opposite direction, though. Could have both, as aliases for one another? 🤷🏽